docs/learning/getting-started/rba/runner-setup.md

```diff
@@ -0,0 +1,36 @@
+---
+
+next:
+  link: '/learning/getting-started/rba/node-setup.md'
+  text: "Add and connect to a remote node through the Runner"
+
+---
+
+# Create a Runner to connect to remote nodes
+
+[Runners](/administration/runner/runner-intro.html) are a recent addition to the PagerDuty Process Automation and Runbook Automation architecture. They are primarily used to automate tasks in secure, remote environments where direct access to infrastructure and services is restricted. This is accomplished by using a "reverse proxy" architecture. Runners regularly query (outbound only) the Runbook Automation instance for tasks. Runners are assigned to specific  projects, and then [jobs](/learning/getting-started/jobs/what-is-a-job.html) within those projects can be sent to the Runner to execute in the remote environment.  
```

Comment:
For links it's better to point them at the ".md" than ".html".

We've seen bugs with .html not parsing the base directory properly.
(applies to all pages)

Reply:
Good catch.  I'll go back and revise.

Reply:
I've fixed all of the links that used .html and converted them to .md so I think we're good to go.